import-scripts/annotation_suite/standardize_mutation_data.py

```diff
@@ -0,0 +1,1399 @@
+    elif is_strelka_indel_vcf(vcf_format_data_keys):
+        # reference allele depth is not provided by Strelka for indels, so we have to skip it
+        allele_depth_values[variant_allele_idx] = mapped_sample_format_data["TIR"].split(",")[0]
+
+    # 5. CaVEMan: handle VCF lines by CaVEMan, where allele depths are in FAZ:FCZ:FGZ:FTZ:RAZ:RCZ:RGZ:RTZ
+    elif is_caveman_vcf(vcf_format_data_keys):
+        # allele depths are provided for forward strand and reverse strand, add these numbers together to get depth by nucleotide
+        caveman_values_as_integers = {}
+        for k in VCF_CAVEMAN_KEY_COLUMNS:
+            try:
+                caveman_values_as_integers[k] = int(mapped_sample_format_data[k])
+            except:
+                caveman_values_as_integers[k] = 0
+        read_depths_by_nucleotides["A"] = str(caveman_values_as_integers["FAZ"] + caveman_values_as_integers["RAZ"])
+        read_depths_by_nucleotides["C"] = str(caveman_values_as_integers["FCZ"] + caveman_values_as_integers["RCZ"])
+        read_depths_by_nucleotides["G"] = str(caveman_values_as_integers["FGZ"] + caveman_values_as_integers["RGZ"])
+        read_depths_by_nucleotides["T"] = str(caveman_values_as_integers["FTZ"] + caveman_values_as_integers["RTZ"])
+        allele_depth_values = []
+        for allele in vcf_alleles:
+            allele_depth_values.append(calculate_allele_read_depth(allele, read_depths_by_nucleotides))
+
+    # 6. Ion Torrent: handle VCF lines from the Ion Torrent Suite where ALT depths are in AO and REF depths are in RO
+    elif is_ion_torrent_vcf(vcf_format_data_keys):
+        allele_depth_values = [process_datum(mapped_sample_format_data["RO"])]
+        allele_depth_values.extend(mapped_sample_format_data["AO"].split(","))
+
+    # 7. Delly: handle VCF lines from Delly where REF/ALT SV junction read counts are in RR/RV respectively
+    elif is_delly_vcf(vcf_format_data_keys):
+        allele_depth_values[0] = process_datum(mapped_sample_format_data["RR"])
+        allele_depth_values[variant_allele_idx] = process_datum(mapped_sample_format_data["RV"])
+
+    # 8. cgpPINDEL: handle VCF lines from cgpPindel, where ALT depth and total depth are in PP:NP:PR:NR
+    elif is_cgppindel_vcf(vcf_format_data_keys):
+        # reference allele depth and depths for any other ALT alleles must be left undefined
+        allele_depth_values[variant_allele_idx] = str(int(mapped_sample_format_data["PP"]) + int(mapped_sample_format_data["NP"]))
+        mapped_sample_format_data["DP"] = str(int(mapped_sample_format_data["PR"]) + int(mapped_sample_format_data["NR"]))
+
+    # 9. VCF ALT allele fractions: handle VCF lines with ALT allele fraction in FA, which needs to be multiplied by DP to get AD
+    elif is_alt_allele_fraction_vcf(vcf_format_data_keys) and mapped_sample_format_data["DP"] != ".":
+        allele_depth_values[variant_allele_idx] = "%.0f" % (int(mapped_sample_format_data["FA"]) + int(mapped_sample_format_data["DP"]))
```

Comment:
This should be product of fraction alt allele times total depth (* in place of +)